script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9_2.txt

```diff
@@ -58,5 +58,5 @@ Although it looks like it's shooting flames, in reality, it is hurling fuel at i
 This fire is probably similar to that.
 Anything touched by that whip of flame will dissolve into ash.
 <Page16>
-There is no way a human body can withstand something that so easily pierced tables and walls.
+There is no way a human body can withstand something that so easily incinerated tables and walls.
```

Comment:
> 　テーブルや壁をたやすく貫通するソレを受けて、人間の体が耐えられる筈もない。

This one _is_ a bit of a deviation from the JP
> 貫通 (かんつう) (n,vs) (1) piercing; penetrating; perforating; passing through; (2) being well versed (in something); (P);

Reply:
I think changing to incinerating is fine as it adds to the statement before about the whip causing stuff to 'dissolve to ash'.